recipe/vina.patch

```diff
@@ -101,19 +103,89 @@ index cf9d426..51609bd 100644
  clean:
  	rm -f *.o
 diff --git a/build/python/setup.py b/build/python/setup.py
-index 938732e..28b3d98 100644
+index b182fda..81e3347 100644
 --- a/build/python/setup.py
 +++ b/build/python/setup.py
-@@ -297,8 +297,6 @@ class CustomBuildExt(build_ext):
-         
+@@ -45,7 +45,7 @@ def in_conda():
+ 
+ def find_version():
+     """Extract the current version of AutoDock Vina.
+-    
++
+     The version will be obtained from (in priority order):
+     1. version.py (file created only when using GitHub Actions)
+     2. git describe
+@@ -56,7 +56,7 @@ def find_version():
+     if os.path.isfile(version_file):
+         with open(version_file) as f:
+             version = f.read().strip()
+-        
++
```

Comment:
Please avoid these spurious changes in the patch

Reply:
those are added automatically upon saving the file - it's not super practical to remove those but i disabled the option for now.